samples/languages/python/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+export PATH="../../../tools:$PATH"
```

Comment:
What's the rational behind replacing Makefile with `build.sh` and `run.sh`?

Reply:
Makefiles are too complicated for samples. `build.sh` and `run.sh` are just for CI purposes (eventually), and maybe we can get rid of them as well for all samples that follow the same pattern.

Reply:
I removed them. We can think about it again once CI for samples is added.